api.yaml

```diff
@@ -987,6 +987,39 @@ components:
           type: string
         valid:
           type: boolean
+    SetAccountOwner:
+      type: object
+      properties:
+        accountId:
+          type: string
+          pattern: ^(\d{12})$
+        name:
+          type: string
```

Comment:
Are we adding `name` to the AccountOwner object as well?

Reply:
Oops, no. Good catch! I'll remove that.